mergify_engine/tests/functional/actions/test_queue.py

```diff
@@ -3007,6 +3008,174 @@ async def test_queue_manual_merge(self):
+        await self.run_engine()
+
+        ctxt_p_merged = context.Context(self.repository_ctxt, p_merged)
+        q = await merge_train.Train.from_context(ctxt_p_merged)
+
+        # my 3 PRs + 2 merge-queue PR
+        pulls = await self.get_pulls()
+        assert len(pulls) == 5
+
+        tmp_mq_p1 = pulls[1]
+        tmp_mq_p2 = pulls[0]
+        assert tmp_mq_p1["number"] not in [p1["number"], p2["number"], p3["number"]]
+        assert tmp_mq_p2["number"] not in [p1["number"], p2["number"], p3["number"]]
+
+        await self._assert_cars_contents(
+            q,
+            p_merged["merge_commit_sha"],
+            [
+                TrainCarMatcher(
+                    [p1["number"]],
+                    [],
+                    p_merged["merge_commit_sha"],
+                    "created",
+                    tmp_mq_p1["number"],
+                ),
+                TrainCarMatcher(
+                    [p2["number"]],
+                    [p1["number"]],
+                    p_merged["merge_commit_sha"],
+                    "created",
+                    tmp_mq_p2["number"],
+                ),
+            ],
+        )
+
+        # Change the configuration and introduce disallow_checks_interruption_from_queues
+        updated_rules = copy.deepcopy(rules)
+        updated_rules["queue_rules"][0]["disallow_checks_interruption_from_queues"] = [
+            "default"
+        ]
+        p_new_config = await self.create_pr(
+            files={".mergify.yml": yaml.dump(updated_rules)}
+        )
+        await self.merge_pull(p_new_config["number"])
+        await self.wait_for("pull_request", {"action": "closed"})
+        await self.run_engine()
+        p_new_config = await self.get_pull(p_new_config["number"])
+
+        ctxt_p_new_config = context.Context(self.repository_ctxt, p_new_config)
+        q = await merge_train.Train.from_context(ctxt_p_new_config)
+
+        # my 3 PRs + 2 merge-queue PR
+        pulls = await self.get_pulls()
+        assert len(pulls) == 5
+
+        tmp_mq_p1 = pulls[1]
+        tmp_mq_p2 = pulls[0]
+        assert tmp_mq_p1["number"] not in [p1["number"], p2["number"], p3["number"]]
+        assert tmp_mq_p2["number"] not in [p1["number"], p2["number"], p3["number"]]
+
+        await self._assert_cars_contents(
+            q,
+            p_new_config["merge_commit_sha"],
+            [
+                TrainCarMatcher(
+                    [p1["number"]],
+                    [],
+                    p_new_config["merge_commit_sha"],
+                    "created",
+                    tmp_mq_p1["number"],
+                ),
+                TrainCarMatcher(
+                    [p2["number"]],
+                    [p1["number"]],
+                    p_new_config["merge_commit_sha"],
+                    "created",
+                    tmp_mq_p2["number"],
+                ),
+            ],
+        )
+
+        # Put second PR at the begining of the queue via pr priority Checks
```

Comment:
```suggestion
        # Put third PR at the begining of the queue via pr priority Checks
```
it is the third right ? otherwise LGTM

Reply:
I will fix that into another PR